### DISCUSSION TITLE
Virtual Power Plant

### DISCUSSION THREAD

**Discussion by cwagz** [jasonacox/Powerwall-Dashboard#65] [General]
We got an invite through the Tesla app. Looks like I will be grid charged at 5am and then sell for $2kwh during the event tomorrow! (If I understand correctly)

![Screenshot 2022-08-16 at 8 04 01 PM](https://user-images.githubusercontent.com/13752647/185025591-c69116a2-2cf1-40f0-a4a2-f98c40de3e90.png)

**Comment by jasonacox** [comment]
Nice @cwagz !  Let us know how it goes... graphs please! 😀

**Reply by cwagz** [reply]
My system charged as normal from the sun. The difference was where normally I would be running on batteries 100% from 4pm-9pm (TOU) the system waited until the event started at 6pm and began discharging the batteries at 11.5kw to the grid. Whatever we used was subtracted from this. We hit our 20% backup reserve at about 7:45pm. I guess the $2 per kw is for what you send out extra compared to a normal day. It should be interesting to see how this pans out money wise. Tesla says we should get a check in March 2023 for any participation until October 31st this year.

I do wonder if the extra wear and tear on the batteries is worth it. With the AC running the system has been going from 100% to 20% all summer but definitely not at this rate of discharge.

Virtual Power Plant event (8/17/22):
![image](https://user-images.githubusercontent.com/13752647/185291377-ac421491-d06b-4a3e-a3b1-3aeb24757d70.png)

Normal Day with some clouds (8/16/22):
![image](https://user-images.githubusercontent.com/13752647/185291543-d9e14949-664e-44f5-a727-bb81acc6ce81.png)

Right before my system hit 20%:
![image](https://user-images.githubusercontent.com/13752647/185291782-f92eb511-442b-404e-912b-326faddd39c6.png)

Near end of event (lots of people hit their reserve):
![image](https://user-images.githubusercontent.com/13752647/185291843-6a0810d2-af6f-47aa-8688-56beef9fade3.png)

**Reply by jasonacox** [reply]
WOW!  That's wild @cwagz !  Thanks for sharing! I like how they show the community so you really feel like you are contributing to something bigger.  Of course, the $2/kWh isn't bad either. That probably helps with the rest of the TOU (7:45pm-9pm) that you are paying a premium for the grid.  I wonder if you selected "time-based" versus "self-powered" if it would reserve power for the TOU window only. That may be how you would optimize for the best payout. But, I'm with you on the wear question.  I actually notched my reserve up to 30% and noticed that the battery degradation leveled out substantially.  Here is a graph of it (was at 20% self-powered, 1st annotation line is 30% and time-based, 2nd is 30% and self-powered).  Both PW's show the same, but I still wonder if it was a fluke just related to them being new and hitting the heavy summer season. I'll probably switch back to 20% soon to test it.

<img width="945" alt="image" src="https://user-images.githubusercontent.com/836718/185334429-c4446722-28e4-460a-986a-c83a90eaca14.png">

It would be interesting to see if you saw a hit on your capacity during the event.  Probably not enough to notice, or just this one event, but still interesting.  

By the way, my graphs look like yours here in SoCal... LOTS of A/C power drain.  I don't know about you, but we suddenly have a lot of our friends reaching out to us with solar questions now that they are getting $1000/mo bills. 💰  It does make you apprecaite your solar system even more. 😀

**Reply by mcbirse** [reply]
Thanks for sharing this @cwagz - it is really interesting!

I haven't joined any VPP here in Sydney but had been thinking about it... several energy companies now offer VPP plans (including Tesla in Aus directly). Lots to think about and consider here though, especially considering longevity of the battery etc. and how VPP use may impact that? I think I prefer to have full control myself at the moment....

@jasonacox - same here regarding ridiculous bills 💰 many friends now asking lots of questions as well. Electricity and gas prices have gone through the roof! So thankful now for my 10kW solar + Powerwall system, installed 16 mths ago... perfect timing, has saved me a boatload already and payoff will be shorter than I thought. I'd love to say I predicted what was going to happen with energy prices, but it was just fluke timing really - I'll take it though. 😃

**Reply by cwagz** [reply]
@mcbirse - Tesla does a nice job in the app of making you feel like you are still in control. You can skip an event, and opt out of the program very easily. I don’t think I would have signed up if it felt more controlling. My biggest fear is the rapid discharge during an event. I am still not sure I will keep participating, especially if these become weekly occurrences. 

@jasonacox - definitely feeling like we had great timing money wise. We used to run the house at 82F all summer just to keep the bills below $600 a month. Now I am set at 79 and everyone is way more comfortable. I don’t think the AC would shut off if we went any lower. Need new windows and door seals I guess!

I have been using the TOU mode since it was available. I still discharge from 100% to 20% each day when the AC is used. Our baseline usage is just too high for two batteries. The change with the event day was that Tesla kept the house running on Solar from 4-6pm. Normally we would be 100% on battery and exporting solar during the high priced time. It does almost look like it may be a wash due to not covering the entire TOU window.

**Reply by jasonacox** [reply]
Good insight @cwagz  - thanks for sharing these details.  And yes, enjoying much cooler home, and happier family, thanks to our solar system. 🌞 

@mcbirse I agree on the payout.  It seems we are ahead of the curve too, especially with the rising energy rates.  I really need to figure out a way to adjust the "savings" graphs to map better to the TOU rate differences.
